extensions/admob/template/android/libadmob/src/com/cocos/admob/proto/openapp/AppOpenAdFullScreenContentCallbackNTF.java

```diff
@@ -0,0 +1,16 @@
+package com.cocos.admob.proto.openapp;
```

Comment:
Similar to my previous comment - can we rename the package name from `openapp` to `appopen`? (And change the directory name as well)